anagrafica/models.py

```diff
@@ -1864,7 +1866,9 @@ def richiedi(self):
         self.autorizzazione_richiedi_sede_riferimento(
             self.persona,
             INCARICO_GESTIONE_TRASFERIMENTI,
-            invia_notifica_presidente=True
+            invia_notifica_presidente=True,
+            auto=Autorizzazione.AP_AUTO,
+            scadenza=settings.AUTORIZZAZIONE_AUTOMATICA,
```

Comment:
Possiamo rendere esplicita la costante, piuttosto che fare affidamento sul default? In particolare non e' necessario sia configurabile per istanza (si tratta di un regolamento CRI), quindi direi che il metodo migliore sia usare una costante di Trasferimento:

```python
class Trasferimento(...):
  APPROVAZIONE_AUTOMATICA = timedelta(days=30)

...

self.autorizzazione_richiedi_sede_trasferimento(...
    scadenza=Trasferimento.APPROVAZIONE_AUTOMATICA, ...
```